.github/pull_request_template.md

```diff
@@ -8,6 +8,6 @@
 
 ## Checks
 
-- [ ] updated changelog
-- [ ] updated documentation
-- [ ] updated tests 
+  - [ ] updated changelog
+  - [ ] updated documentation
+  - [ ] updated tests
```

Comment:
Codacy has a fix for the issue: [Expected: 0; Actual: 2](https://app.codacy.com/gh/Mjolnir-Forge/mjolnir-core/pullRequest?prid=9403582)
```suggestion
- [ ] updated tests
```